lib/public/views/Runs/format/formatDetectorQuality.js

```diff
@@ -10,17 +10,31 @@
  * granted to it by virtue of its status as an Intergovernmental Organization
  * or submit itself to any jurisdiction.
  */
-import { h, iconBan, switchCase } from '/js/src/index.js';
+import { getValueFromCssVar, isContrastBlack } from '../../../components/common/badge.js';
+import { h, iconBan } from '/js/src/index.js';
+
+const OnlineGeneralQualityColors = {
```

Comment:
Could you create a generic `Colors` enum? And in the same file export a function `getContrastColor` that takes a color, and either directly return its contrast if it's raw color or compute it using CSS if it's a CSS var.
The objective is that the CSS leak that we are doing is contained in only one file.

Reply:
applied

Reply:
done